### PR TITLE
EID-1412: Add a static error page for Proxy Node

### PR DESCRIPTION
**Pull request by CharlesIC**
A static error page to which the eIDAS proxy node can redirect users in case of an unrecoverable error.
This is basically the same as the _Something went wrong_ page but with the man text specific to eIDAS. The text is as designed by James Butler in EID-1229

Screen Shot 2019-03-21 at 17 18 24

### CODE REVIEW

File: app/models/feedback_source_mapper.rb [hunk 1]
```diff
@@ -18,6 +18,7 @@ def initialize(product_page_url)
       'FAILED_SIGN_IN_PAGE' => 'failed_sign_in',
       'FAILED_COUNTRY_SIGN_IN_PAGE' => 'failed_country_sign_in',
       'EIDAS_SCHEME_UNAVAILABLE' => 'redirect_to_country',
+      'PROXY_NODE_ERROR' => 'proxy_node_error',
```
Comment (jakubmiarka) [inline app/models/feedback_source_mapper.rb]: The convention here is to append `_PAGE`

Comment (CharlesIC) [reply app/models/feedback_source_mapper.rb]: Fine... Done

File: config/locales/cy.yml [hunk 2]
```diff
@@ -640,6 +641,8 @@ cy:
       heading: Mae’n ddrwg gennym, mae rhywbeth wedi mynd o’i le
       error_message: Gall hyn fod oherwydd bod eich sesiwn wedi’i amseru allan neu roedd gwall yn y system.
       start_again: Oherwydd bod hwn yn wasanaeth diogel, bydd angen i chi ddechrau eto.
+    proxy_node_error:
+      start_again: Because you were using a secure service from another European country, you’ll need to go back to the page where you started using that service.
```
Comment (tomrosier) [inline config/locales/cy.yml]: Has the Welsh been requested?

Comment (CharlesIC) [reply config/locales/cy.yml]: Yep, doing the spreadsheet now 👍

**Pull request merged**